rj_gameplay/rj_gameplay/action/pivot.py

```diff
@@ -13,20 +13,36 @@
 
 class Pivot(action.IFiniteAction):
 
-    """
-    Pivot Skill
-    """
-    def __init__(self, robot_id : int, pivot_point: np.ndarray, target_point: np.ndarray, priority : int = 1):
+    def __init__(self, robot_id:int, pivot_point:np.ndarray, target_point:np.ndarray, dribble_speed:float=1.0, priority:int=1):
```

Comment:
Same complaints about default arguments

Reply:
I'm going to leave priority, because I don't any skill use it currently, so probably something for after comp.